ui/core/components/individual_sim_ui/apl_actions.ts

```diff
@@ -582,6 +583,14 @@ const actionKindFactories: { [f in NonNullable<APLActionKind>]: ActionKindConfig
 		newValue: () => APLActionCancelAura.create(),
 		fields: [AplHelpers.actionIdFieldConfig('auraId', 'auras')],
 	}),
+	['setAuraStacks']: inputBuilder({
+		label: 'Set Aura Stacks',
+		submenu: ['Misc'],
+		shortDescription: 'Activates and sets the stacks of an aura.',
```

Comment:
We have a similar action in the SoD repo. Could we use consistent naming for it here?

![image](https://github.com/user-attachments/assets/32b33f40-25d1-4ef4-b847-6fee9474a34b)

Reply:
yep, will do!